### DISCUSSION TITLE
fatal: this operation must be run in a work tree

### DISCUSSION THREAD

**Discussion by emarashliev** [community/community#25888] [Actions]
<p>Hey guys,</p>
<p>I’m geting this very annoying error when I run a GitHub action, I’m running the same steps in a Docker container without have this error.  </p>
<p><a href="https://github.com/nodes-vapor/submissions/commit/77438895cb1e2b0b3e8f8a79178c643360024103/checks" rel="nofollow noopener">Here is the error</a></p>
<p><a href="https://github.com/nodes-vapor/github-actions/tree/develop/actions/jazzy-docs" rel="nofollow noopener">Here is a the action</a></p>

**Comment by ethomson** [comment, marked as answer]
<p>That error message comes from git:  it occurs when you run git in a directory that’s not a working tree (ie is a checked out repository).  Looking at your workflow, it looks like you’ve got successful runs now so it looks like you might have fixed your workflow.</p>
<p>Is that correct?  Or are you still facing this problem?</p>

**Comment by emarashliev** [comment]
<p>Yes, it was some fail in the bash script. I’m sorry just forgot to mark it as resoved issue. But I have other quesion - why <a href="https://github.com/nodes-vapor/submissions/blob/master/.github/workflows/release.yml" rel="nofollow noopener">this</a> Action trigers 3 jobs? Is that somthing usual?</p>

**Comment by chrispat** [comment]
<p>GitHub events have multiple activity types associated please see <a href="https://help.github.com/en/articles/events-that-trigger-workflows#release-event-release" rel="nofollow noopener">https://help.github.com/en/articles/events-that-trigger-workflows#release-event-release</a> for detials on how to only trigger for a single activity type on an event.</p>

**Comment by emarashliev** [comment]
<p>I see, but your documentation is really vague, is not clear how this should be declared?</p>
<p>like this:</p>
<pre><code>on:
  release:
    types: [published]
      name: Auto docs
      jobs:...
</code></pre>
<p>like that:</p>
<pre><code>on:
  release
    types: [published]
    name: Auto docs
    jobs:..
</code></pre>
<p> As it described here <a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#onevent_nametypes" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions#onevent_nametypes</a></p>
<p>it fails with: </p>
<pre><code>No jobs defined in jobs
</code></pre>
<p>This one here <a href="https://help.github.com/en/articles/events-that-trigger-workflows#release-event-release" rel="nofollow noopener">https://help.github.com/en/articles/events-that-trigger-workflows#release-event-release</a></p>
<p>also doesn’t make any sense</p>
<pre><code>on:
  project_column:
    types: [published]
</code></pre>
<p>What is “project_column:”?</p>
<p>It’s very annoying because I have to guess, try and wait every time …</p>
